src/data/finsupp/basic.lean

```diff
@@ -2483,6 +2479,11 @@ instance [semiring R] [add_comm_monoid M] [module R M] : module R (α →₀ M)
 
 variables {α M} {R}
 
+lemma comap_domain_smul {R : Type*} [add_monoid M] [monoid R] [distrib_mul_action R M]
```

Comment:
This probably makes more sense slightly lower, underneath `map_domain_smul`

Reply:
Done.  I moved it at the end of that section, since the earlier `comap_domain` lemmas followed all the analogous `map_domain` lemmas in block, rather than interleaved.